spackbot/pr_add_reviewers.py

```diff
@@ -211,16 +280,6 @@ async def add_reviewers(gh, repository, pull_request, number):
             )
             await gh.post(pull_request["comments_url"], {}, data={"body": comment_body})
 
-        # Ask people to maintain packages that don't have maintainers.
-        if unmaintained_pkgs:
-            # Ask for maintainers
-            # https://docs.github.com/en/rest/reference/issues#create-an-issue-comment
-            comment_body = no_maintainers_comment.format(
-                author=pull_request["user"]["login"],
-                packages_without_maintainers="\n* ".join(sorted(unmaintained_pkgs)),
-            )
-            await gh.post(pull_request["comments_url"], {}, data={"body": comment_body})
```

Comment:
Why is this removed?

Reply:
It’s just moved. See above. It’s actually working: https://github.com/spack/spack/pull/24561